src/components/navigation/NavigationBar.tsx

```diff
@@ -51,41 +47,50 @@ export default function NavigationBar(): ReactElement {
   return (
     <header id="navigation-header" className="navigation-header">
       <div className="navigation-header-container">
-        <Logo title="PromoPal" className="logo" height={50} />
+        <Link to={Paths.Home}>
+          <Logo title="PromoPal" className="logo" height={50} />
+        </Link>
```

Comment:
If you make the window **width** smaller, it collapses the menu items. That's correct, but if I then make the width larger, they do no uncollapse (expand). Is this happening for you too?

Reply:
Oh yeah you're right it happens for me too. Ahh I think that might be something to do with ant design... I'll have to figure this out another time I think :(

Reply:
Ah okay no worries